src/components/AdminSettings/RecordingServers.vue

```diff
@@ -150,6 +157,11 @@ export default {
 		this.uploadLimit = parseInt(state.uploadLimit, 10)
 	},
 
+	mounted() {
```

Comment:
We need to provide an updated value, if we add/remove a signaling server in another component. Otherwise an admin should reload the page to unlock recording =)
I could take it from here, if you want

Reply:
Yes, see my comment :-) 
Same in SIP currently. I would be happy if you can take it from here, as I fail to see an easy way to do that (without creating a store or using props in the parent component).